browser/ui/BUILD.gn

```diff
@@ -57,8 +57,13 @@ source_set("ui") {
     "//brave/components/brave_shields/core/browser",
     "//brave/components/webcompat/core/common",
     "//chrome/browser/ui:browser_navigator_params_headers",
+    "//chrome/browser/ui/tabs:tabs_public",
   ]
 
+  if (!is_android) {
+    deps += [ "//brave/browser/ui/tabs:impl" ]
```

Comment:
It seems to work adding this here, but I wasn't sure if I add it to `brave_chrome_browser_deps` instead?

Reply:
we should also add a brave subclass for the android tab features, but yes you should also add to `brave_chrome_browser_deps`

Reply:
Cool, should have android support now :smile: